mission-control/docs/playbooks/concepts/artifacts.md

```diff
@@ -0,0 +1,16 @@
+# Artifacts
```

Comment:
This should probably have details on how to setup the artifact store and example of using it